src/views/Users/UserDetails.vue

```diff
@@ -29,16 +21,6 @@
             >
           </div>
 
-          <transition>
-            <base-card v-if="isResourceTableVisible">
-              <resource-availability-table
-                :tableColumns="tableColumnsResources"
-                :resourceItem="resourceItemResources"
-                :user="user"
-              ></resource-availability-table>
-            </base-card>
-          </transition>
```

Comment:
We may keep this. We can still see what the user has bought. We need to use a different endpoint for it though:
`GET /resources/availability/purchased/{userId}`